unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/ExpressionsHUD/ExpressionsHUDView.cs

```diff
@@ -32,8 +32,20 @@ private void Awake()
     {
         openExpressionsDelegate = (x) => { if (IsVisible()) ToggleContent(); };
         openExpressionsAction.OnTriggered += openExpressionsDelegate;
-        hideContentButton.onClick.AddListener(HideContent);
         showContentButton.onClick.AddListener(ToggleContent);
+
+        for (int i = 0; i < hideContentButtons.Length; i++)
+        {
+            hideContentButtons[i].onClick.AddListener(HideContent);
+        }
+    }
+
+    private void Update()
+    {
+        if (Input.GetKeyDown(KeyCode.Escape))
```

Comment:
can this be done using the InputController?

Reply:
> can this be done using the InputController?

Good point. I'll check that out

Reply:
awsome thx!